operation/opid.go

```diff
@@ -137,7 +137,21 @@ func (m *Manager) createWithID(ctx context.Context, id uuid.UUID, method string,
 
 	o := m.Find(id)
 	if o != nil {
-		m.logger.CWarnw(ctx, "attempt to create duplicate operation", "id", id.String(), "method", method)
+		// Given the exceedingly low chance of a randomly generated UUID colliding, unless it was purposeful, the only way
+		// opids will collide is if an operation goes to a module and then back to a dependency as a child operation. In those cases,
```

Comment:
I'm having trouble understanding this case; do you mean something like `SetPower` on a modular base eventually being routed with the same operation ID as `SetPower` to a motor? How was this issue even manifesting? Just these `Warn` logs appearing in that situation (looks like the existing op would also get overwritten)?

Reply:
yes, basically - `SetPower` on a modular base calling into `SetPower` on a builtin motor will trigger the warning and also overwrite the existing op. Happy to workshop the wording a bit, let me know your thoughts on improving the comment

Reply:
Ah could you maybe just add the concrete example as a parenthetical `and then back to a dependency as a child operation (e.g. SetPower on a modular base calling into SetPower on a builtin motor)`?